tests/dummy/mirage/factories/file.js

```diff
@@ -1,12 +1,22 @@
 import { faker } from "@faker-js/faker";
 import { Factory } from "miragejs";
 
+function checksum() {
+  // https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript
+  const str = Array.from(crypto.getRandomValues(new Uint8Array(32)), (dec) =>
+    dec.toString(16).padStart(2, "0"),
+  ).join("");
+
+  return `sha256:${str}`;
```

Comment:
Both of these seem simpler:
https://fakerjs.dev/api/git.html#commitsha
https://stackoverflow.com/questions/9407892/how-to-generate-random-sha1-hash-to-use-as-id-in-node-js
We can also just leave it, as its just mock data

Reply:
How the hell did I not find this in the faker docs?? :joy: